### DISCUSSION TITLE
Deleting nodes in task graph / reverting back one step in task graph

### DISCUSSION THREAD

**Discussion by dharik-arsath** [dask/dask#7898] [Q&A]
1) How to delete one Node in task graph, to overcome the error

I got some error while feature transformation step, now I need to delete that node from the task graph how can I do that?

**Comment by jsignell** [comment]
You might be able to use [graph manipulation](https://docs.dask.org/en/latest/graph_manipulation.html) but generally people recreate the task graph from scratch rather than altering existing graphs.
